scripts/blog.js

```diff
@@ -0,0 +1,94 @@
+// Defines footer
+const footer = document.getElementById("blog-footer");
+
+// Creates title of blog
+const title = document.createElement("h1");
+title.textContent = "Adoptadog blog";
+title.id = "main-content";
+title.className = "blog-title";
+document.body.insertBefore(title, footer);
+
+// Creates blog container
+const container = document.createElement("div");
+container.className = "blog-container";
+document.body.insertBefore(container, footer);
+
+// Creates elements for first blog
```

Comment:
Great work, Now that you have the loop set up could you refactor this code work with it as well?